src/plugins/intel_gpu/src/graph/include/deconvolution_inst.h

```diff
@@ -60,6 +60,23 @@ class typed_primitive_inst<deconvolution> : public typed_primitive_inst_base<dec
     static layout calc_output_layout(deconvolution_node const& node, kernel_impl_params const& impl_param);
     static std::string to_string(deconvolution_node const& node);
 
+    bool need_reset_input_memory() const override {
+        auto input_layout = _deps[0].first->_impl_params->get_output_layout(0);
```

Comment:
Do we really need this for deconvolution? As I can see, only convolution and binary_convolution use physical paddings for calculation, isn't?

Reply:
I am not sure exact kernel impl of deconv, but if there is predefined output size, it may need. We can optimize out this afterwards if it is known to be not needed.